### DISCUSSION TITLE
Another nextjs app redirection affecting newly created nextjs app

### DISCUSSION THREAD

**Discussion by einer20** [vercel/next.js#37580] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:22 PDT 2022; root:xnu-8020.121.3~4/RELEASE_X86_64
    Binaries:
      Node: 16.10.0
      npm: 8.5.3
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.1.7-canary.31
      react: 18.1.0
      react-dom: 18.1.0

### What browser are you using? (if relevant)

Chrome

### Describe the Bug

I have an app that redirects from the root directory `/` to the `/desk` directory. The issue is whenever I create a new nextjs app and try to access to the root directory i also get redirected to the `/desk` directory. 

The error used to go away when I open chrome on incognito mode, but now even on incognito mode still redirecting to the `/desk` directory even though they are 2 entirely different applications.

Note that it does not happen on windows, only on MAC.

### Expected Behavior

When creating a new app it should not redirect when accessing the root directory.

### To Reproduce

I can't reproduce it. Only happens on my MAC, is not happening on windows.

**Comment by icyJoseph** [comment]
Reload the page with `shift+cmd+r`.

When you do redirects with permanent set to true, it is the browser that no longer desires to go to `localhost:3000/` but instead go to `localhost:3000/desk`

You can also open the devTools, and with the devTools open go to the "reload page" button (usually top left, before the URL bar), click and hold it down, until a special menu shows, then take the second option.

**Reply by tharzeez-ddrobo** [reply]
i had to empty cache and hard reload the tab, right clicking on the refresh icon in chrome opens this menu

**Comment by einer20** [comment]
@icyJoseph Still does not work. I created a new js app just now and when i open the `localhost:3000` it redirects to `localhost:3000/desk`. Even on incognito mode redirects.

**Reply by icyJoseph** [reply]
Yeah but you gotta somehow tell the browser to drop that. Try with the devTools open and cmd+shift+r, I was able to reproduce this several times.

This is the danger, so to speak, with permanent redirects. When you switch apps, you gotta refresh the browser page cache.

**Comment by einer20** [comment]
Sorry for my late response but it worked. Thank you!

**Reply by icyJoseph** [reply]
No worries! Permanent redirects are tricky! Please do consider marking as answered! Thanks!

**Comment by bronz3beard** [comment]
this is not answered in my case i tried all refresh options (chrome open devtools right click on refresh icon) deleted.next rebuilt, deleted all caches in browser and deleted the service worker, it seems maybe this is a browser issue not a nextjs issue?? but it is frustrating 😅

**Reply by bronz3beard** [reply]
this is critical to remember "Permanent redirects are tricky" 😂 also in my case i am dealing with rewrites not redirects

**Reply by bronz3beard** [reply]
the quickest solution is work in the incognito window
